web-app/src/components/BookingModal/index.js

```diff
@@ -26,27 +32,38 @@ const BookingModal = props => {
         <StyleContainer>
           {isEventBeingUpdated && (
             <ModalStatusBanner
+              toggleRejectionResponseView={toggleRejectionResponseView}
               userName={booking.title}
               eventStatus={booking.eventStatus}
               eventType={booking.eventType}
               cancelEvent={cancelEvent}
+              rejectionReason={booking.messages ? booking.messages.length > 0 ? booking.messages[0].message : undefined : undefined}
```

Comment:
I appreciate it's inline, but it's a bit of a nested ternary. Any chance you could bring this out into a function and just put the result in there?